src/apps/opening-hours-editor/OpeningHoursEditorEventContent.tsx

```diff
@@ -10,15 +11,25 @@ const OpeningHoursEditorEventContent: React.FC<
   OpeningHoursEditorEventContentProps
 > = ({ eventInput }) => {
   const { event } = eventInput;
+  const { repetition } = event.extendedProps;
+  const isSeries = repetition?.type === "weekly";
+
   return (
     <div
       className="opening-hours-editor-event-content"
       data-cy="opening-hours-editor-event-content"
     >
-      <b>{event.title}</b>
       <div>
-        {extractTime(event.start)} - {extractTime(event.end)}
+        <b>{event.title}</b>
+        <div>
+          {extractTime(event.start)} - {extractTime(event.end)}
+        </div>
       </div>
+      {isSeries && (
+        <div className="opening-hours-editor-event-content__series-icon">
+          <img src={logo} alt="" className="" />
```

Comment:
```suggestion
          <img src={logo} alt="" />
```

As I see it there is no reason to have an empty classname property.

Please consider adding an alt text to the image describing that this is a repeated event. In other words: This is not a decorative image.